themes/komitywa/templates/base.html

```diff
@@ -0,0 +1,7 @@
+{% extends "!simple/base.html" %}
+
+{% block head %}
+{{ super() }}
+   <link rel="stylesheet" href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/css/{{ CSS_FILE }}" />
```

Comment:
`CSS_FILE` should be defined in pelican configuration and after that line below will duplicate exactly what this line do.